src/htm/algorithms/TemporalMemory.hpp

```diff
@@ -99,6 +99,11 @@ class TemporalMemory : public Serializable
    * @param predictedSegmentDecrement
    * Amount by which segments are punished for incorrect predictions.
    *
+   * Note: A good value is just a bit larger than
+   * (the column-level sparsity * permanenceIncrement). So, if column-level
```

Comment:
OT: #279 it would be nice to reduce the number of parameters for the algorithms, especially in cases like this where a good-enough value can be derived from other settings. Do you think this would be a reasonable candidate to hard-code? 
Pros: 
- less space for user to mess up
- faster parm optimizations
Cons: 
- removes a bit of configurability

Reply:
I agree, but here are two more potential issues:
1) Is this advice always good?  I think it would need further investigation before we remove the parameter in favor of this heuristic.
2) We would have to modify every instance of the TM to omit this parameter.  It would be a lot of work.